File: IT/1/Stringhe/Esercizi/Python/esercizio7.py
# file esercizio7.py
# -*- coding: utf-8 -*-

from __future__ import print_function

s = "3.14159"

t = s.split(".")

print("La parte intera di {} è {}, mentre la parte decimale è 0.{}.".format(s, t[0], t[1]))

File: IT/1/Stringhe/Esercizi/Java/Esercizio7.java
/* file Esercizio7.java */

public class Esercizio7 {
	public static void main(String [] args) {
		String s = "3.14159";
		String [] t = s.split("\\.");

		System.out.println("La parte intera di " + s + " è " + t[0] + ", mentre la parte decimale è 0." + t[1] + ".");
	}
}

